storm-core/src/jvm/org/apache/storm/trident/operation/builtin/Debug.java

```diff
@@ -40,7 +42,7 @@ public Debug(String name) {
 
     @Override
     public boolean isKeep(TridentTuple tuple) {
-        System.out.println(name + tuple.toString());
+        System.out.println("<"+new Date()+"> "+name + tuple.toString());
```

Comment:
Instead of modifying Debug you could use the peek api where you can specify custom actions.